src/components/ViewEnvironments/Drawer/index.tsx

```diff
@@ -46,13 +50,30 @@ function EnvironmentDrawer({ env, onClose }: DrawerParams) {
   });
   const [selectedTag, setSelectedTag] = useState<string | null>(null);
 
+  const [_n, setName] = useLocalStorage<string>("environments-selectedname", "");
```

Comment:
If the `_n` variable is unused it can be omitted from the array destructuring (leave the comma alone, however):

```javascript
  const [, setName] = useLocalStorage<string>("environments-selectedname", "");
```

Reply:
Thanks, we missed that knowledge